src/towncrier/_settings/load.py

```diff
@@ -23,28 +24,30 @@ def __init__(self, *args, **kwargs):
 _underlines = ["=", "-", "~"]
 
 
-def load_config_from_options(directory, config):
-    if config is None:
+def load_config_from_options(
+    directory: Optional[str], config_path: Optional[str]
```

Comment:
I guess that this is private API so is ok to rename the argument.

Reply:
Mypy doesn't like one variable being used with two different types, so I had to rename either the argument or the local use. As the argument contains the path of the config file rather than the configuration itself, I figured that would be the best one to rename. And as you say, it's inside a private module, so it should be safe.

By the way, does Towncrier even have a public API in the form of Python modules or is the command line interface the only public interface? The docs don't mention any modules and the docstrings are rather sparse.